client.js

```diff
@@ -12,16 +12,16 @@ imFeelingLucky.addEventListener("submit", imFeelingLuckyFunc);
 
 function googleSearchFunc(e) {
     e.preventDefault();
-    const title = document.getElementById("title");
-    const image = document.getElementById("image");
-    const text = document.getElementById("description");
+    const title = document.getElementById("title"); //
+    const image = document.getElementById("image"); //
+    const text = document.getElementById("description"); //
     // console.log("You clicked the google search button!");
-    axios.get("http://localhost:3000/search/animal/:id")
+    axios.get("http://localhost:3000/search/animal/:id") //
```

Comment:
The `:id` in the get url doesn't make use of any information passed to it, instead you could interpolate the value of a variable that gets information from your form text input.

e.g. `let search= form.textinput.value`
``http://localhost:3000/search/animal/${search}``